test/libsolidity/syntaxTests/inheritance/override/external_turns_public_no_params.sol

```diff
@@ -0,0 +1,7 @@
+contract A {
+    function f() external pure {}
+}
+contract B is A {
+    function f() public pure {
```

Comment:
Should there be a semantics test for this to check that this in fact overrides the function?

Reply:
Added test.